haiku-apps/bedc/bedc-0.5.0.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="A BeOS file sharing client"
+DESCRIPTION="BeDC is a BeOS file sharing client compatible with the Direct \
+Connect protocol. Currently only chatting works, not filesharing/downloading."
+HOMEPAGE="https://github.com/HaikuArchives/BeAIM/"
+COPYRIGHT="2002 BeDC Team"
+LICENSE="MIT"
```

Comment:
Well it's not BSD (3-clause), it's a custom license.

Reply:
Well, when the recipe was added (during GCI 2017) we thought it was a `BSD (3-clause)`:
https://github.com/haikuports/haikuports/blob/716aed6e3450b601879c7a046391f71e37210f1a/haiku-apps/bdirectconnect/bdirectconnect-0.5.0%7Egit.recipe#L7

BTW, I think we should just close this PR, as I wrote here: https://github.com/haikuports/haikuports/pull/3343#issuecomment-436106563

Hmm, in case the license is not a `BSD (3-clause)` maybe we can have it fixed with this PR.

Reply:
I've just had a look. Although the project's `LICENSE` file pretends it is a "BDC license", it is actually a `BSD (3-clause)` license with these cosmetic changes:
* It has this header:
```
----------------------
BDC License
----------------------

```
* "Copyright (c) **\<year\>, \<copyright holder\>** :arrow_right: "Copyright **2002, The BDC team.**"`
* The `*` that precede each of the 3 conditions becomes a `1.`, `2.` and `3.`.
* "this list of conditions and the following disclaimer" :arrow_right: "this list of conditions, and the following disclaimer" (the diff is just a comma)
* `<organization>` :arrow_right: `BeDC team`
* "COPYRIGHT HOLDERS" :arrow_right: "COPYRIGHT HOLDERS **AND CONTRIBUTORS**"
* "THE IMPLIED WARRANTIES OF MERCHANTABILITY" :arrow_right: "THE IMPLIED WARRANTIES OF **TITLE, NON-INFRINGEMENT,** MERCHANTABILITY"
* "IN NO EVENT SHALL **<**COPYRIGHT HOLDER**>** BE LIABLE" :arrow_right: " IN NO EVENT SHALL  THE COPYRIGHT HOLDER**S** **OR CONTRIBUTORS** BE LIABLE`

So the BDC license **is** a `BSD (3-clause)` license.
That's why, imo, we should:
* **keep** `LICENSE="BSD (3-clause)"`
* install an exact copy of the "LICENSE" file in `$dataDir/licenses/packages/b/bdirectconnect/LICENSE`, as I'm suggesting in #3312.

Adding the `LICENSE` file in our repo would be just noise. Are we going to add every instantiation of each BSD license in our repo?

The best solution is to have the recipe copy the license to a dedicated subdirectory.

@bach5000, I think you can keep this PR open for, say, 7 days, and then close it if you did not get any green lights from the contributors who commented #3312.
(You got my green lights but it looks like my opinion on this topic does not count...)

Reply:
Little problem with this approach: the hpkg metadata will still reference "BSD (3-clause)" whereas the package will provide another custom license.
I would add a copy in haiku-apps/bdirectconnect/licenses/BeDC and set LICENSE="BeDC" in the recipe.

Reply:
Sure.
Anyway, there are 2 questions:
a) do we agree that the license in this recipe a `BSD (3-clause)` and not a custom license?
b) do we allow this approach for all new recipes?

I'm not saying we should fix all similar cases. I'm saying we should start a smooth transition: Everytime a recipe has a version bump, if the contributor editing the recipe is willing to examine the pseudo-custom license for that recipe then, imo, we should accept the switch.

Reply:
@fbrosson 

Those aren't cosmetic changes. They're actual meaningful changes to the license text. So yes, this is a custom license.

That said, I'm certainly not a fan of keeping a copy of the license in our repo...

Reply:
> Those aren't cosmetic changes. They're actual meaningful changes to the license text. So yes, this is a custom license.

The differences between the pseudo-custom BDC License and the `BSD (3-clause)` are **_both_** cosmetic **_and_** meaningful. They are cosmetic because if you fill the placeholders in the BSD license with the "values" of the BDC license then you can see that the BDC license is a copy/paste of the BSD license. And, of course, they are meaningful because the most important information resides in these cosmetic changes.

> That said, I'm certainly not a fan of keeping a copy of the license in our repo...

:+1:
That's exactly what my proposal (#3312) addresses. It allows to stop polluting our repo with files that are already in the source tarballs.